### PR TITLE
[MM-41844] Warn cloud admin about workspace deletion at end of trial

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Add a mention into the post-trial email that the workspace will be deleted after 30days if they do not subscribe.

QA question: Is there a way to reduce a trial duration?

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41844

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by nickmisasi — APPROVED** [review]

File: app/email/email_test.go [hunk 1]
```diff
@@ -162,3 +162,41 @@ func TestSendCloudTrialEndWarningEmail(t *testing.T) {
 		verifyMailbox(t)
 	})
 }
+
+func TestSendCloudTrialEndedEmail(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (marianunez) [reply app/email/email_test.go:166]: +1 :rocket:

**Review by marianunez — APPROVED** [review]
Thanks for the test addition 💯

**Comment by marianunez** [comment]
> QA question: Is there a way to reduce a trial duration?

Yes, in Stripe you can click Update Subscription and change when the trial would end. In this case you would want to change it to 4 days from today and you should get the email the next day. Not sure if you get the email immediately if you change it to less than 3 days.

**Comment by JulienTant** [comment]
> > QA question: Is there a way to reduce a trial duration?
> 
> Yes, in Stripe you can click Update Subscription and change when the trial would end. In this case you would want to change it to 4 days from today and you should get the email the next day. Not sure if you get the email immediately if you change it to less than 3 days.

Do we also want this mentioned in the 3 days before the end of the trial email? Because I only changed the email that is sent at the very end of the trial

File: app/email/email_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (JulienTant) [reply app/email/email_test.go:166]: Tests tests tests

**Comment by marianunez** [comment]
> Do we also want this mentioned in the 3 days before the end of the trial email? Because I only changed the email that is sent at the very end of the trial

No, you are right. Got confused with the 3 day left warning email. In this case, then you can Cancel Trial in Stripe and you would get the email immediately

**Review by stevemudie — APPROVED** [review]
Nice work @JulienTant!

This worked as expected.

I do have a rewording suggestion though.

The line that currently reads "_If you have not added your payment information within 30 days…_" would come across better if it read "_If you do not add your payment information within 30 days…_"

CC: @marianunez

**Comment by JulienTant** [comment]
> Nice work @JulienTant!
> 
> This worked as expected.
> 
> I do have a rewording suggestion though.
> 
> The line that currently reads "_If you have not added your payment information within 30 days…_" would come across better if it read "_If you do not add your payment information within 30 days…_"
> 
> CC: @marianunez

Hey @stevemudie ! I'm also adding @jasonblais; he's owning a PR on the documenting using those words: https://github.com/mattermost/docs/pull/5440

**Comment by jasonblais** [comment]
@stevemudie I like your suggestion, let's go with `If you do not add your ...` instead. I'll update the docs PR as well :+1:

**Comment by jasonblais** [comment]
cc @JulienTant forgot to tag you on the above

**Comment by JulienTant** [comment]
@stevemudie @jasonblais Cool, I updated the PR!

**Pull request merged**